School request

Tmfcerqueira opened the pull request:
Hi, my name is Tomás Cerqueira from Portugal, more specific from school INETE (înete-Instituto De Educação Tecnica) [Inete-Institute of Technical Education] and im trying to download a product of JetBrains with a student license but my student email not working, so im requesting if you could accept this request, also my email is I200064@inete.net , if it could help setting the domain etc.. 

Best regards,
Tomás cerqueira.

[File.zip](https://github.com/Akandesh/swot/files/9976248/File.zip)

Tmfcerqueira commented:
[inete.txt](https://github.com/Akandesh/swot/files/10036606/inete.txt)

HumbertoRTS commented:
Hello, I study at Universidad Privada del Norte (private northern university) and I am trying to download a JetBrains product with a student license but my student email is not working so I ask you if you can accept this request, also my email is n00428614@upn.pe, if you could help set up the domain, etc.

[file.zip](https://github.com/user-attachments/files/17638799/file.zip)

gerardocrr committed:
Create lagos.txt

Commit:
Add Centro Universitario del Oriente de Hidalgo

philipto committed:
Merge pull request #39498 from Eder3323/add-centrouniversitariohgo

Pull request accepted.

philipto committed:
Merge pull request #39494 from gerardocrr/master

Pull request accepted.

philipto committed:
Merge pull request #39493 from budi-imam-prasetyo-su/master

Pull request accepted.

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

vunm-597 committed:
add dhv.txt

philipto committed:
Merge pull request #39501 from vunm-597/master

Pull request accepted.

Aniketh896 committed:
Added Indian Institute of Management Sirmaur

AlvaroGhostPlay committed:
My university

My university, located in Mexico, for online students

philipto committed:
Merge pull request #39507 from AlvaroGhostPlay/master

Pull request accepted.

OMR8X committed:
Add files via upload

philipto committed:
Merge pull request #39527 from OMR8X/master

Pull request accepted.

gregov9 committed:
Añadir dominio ujgh.edu.ve

MohammadQaseemQiam committed:
Merge branch 'JetBrains:master' into master

MohammadQaseemQiam committed:
Rename kpu.txt to Kabul Polytechnic University.txt

gregov9 committed:
Add Universidad Dr. José Gregorio Hernández to ujgh.txt

philipto committed:
Merge pull request #39534 from gregov9/patch-1

Pull request accepted.

sadat-southern committed:
Add Southern University Bangladesh domain

gregov9 committed:
Merge branch 'master' into master

philipto committed:
Merge pull request #39539 from sadat-southern/master

Pull request accepted.

philipto committed:
Merge pull request #39532 from gregov9/master

Pull request accepted.

dong-junn committed:
add jeiu university(인천재능대학교)

Froxsen committed:
Add mpg-dortmund domain

Max-Planck-Gymnasium Dortmund

philipto committed:
Renaming file according the repository rules

philipto committed:
Merge pull request #39558 from Froxsen/patch-2

Pull request accepted.

philipto committed:
Merge pull request #39549 from dong-junn/master

Pull request accepted.

MohammadQaseemQiam committed:
Delete kpu.txt

Suliyaa98 committed:
Create ndri.txt

Olga-Sh committed:
Add mirror for Maven Central

dong-junn committed:
add m.jeiu.ac.kr

wannapa12 committed:
Create scsa.txt

philipto committed:
Merge pull request #39572 from wannapa12/master

Pull request accepted.

philipto committed:
Merge pull request #39568 from dong-junn/master

Pull request accepted.

philipto committed:
Merge pull request #39564 from Suliyaa98/master

Pull request accepted.

philipto committed:
Renaming file according the repository rules

philipto committed:
Merge pull request #39421 from AmbrocioIsaias2808/master

Add Instituto Tecnológico de Matamoros (matamoros.tecnm.mx)

iamRaz-01 committed:
Add files via upload

philipto committed:
reorganizing stoplist

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Fraud

philipto committed:
Merge pull request #39596 from iamRaz-01/master

Pull request accepted.

rodrigomart123 committed:
Add AE Prosa (aeprosa.pt)

djpatel1983 committed:
jau.in added

Junagadh Agricultural University

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Fraud

philipto committed:
Merge pull request #39600 from djpatel1983/master

Pull request accepted.

philipto committed:
Merge pull request #39599 from rodrigomart123/master

Pull request accepted.

Emzy123 committed:
Add files via upload

Emzy123 committed:
Delete lib/domains/ng/edu/custech.txt

Emzy123 committed:
Add cuestech.edu.ng for Confluence University of Science and Technology

ericbardetcecc-cmyk committed:
Create ceccharlevoix.txt

philipto committed:
Merge pull request #39610 from ericbardetcecc-cmyk/patch-1

Pull request accepted.

philipto committed:
Merge pull request #39608 from Emzy123/master

Pull request accepted.

Aminev9 committed:
EMAA BUSINESS SCHOOL

Aminev9 committed:
EMAA BUSINESS SCHOOL

Aminev9 committed:
EMAA BUSINESS SCHOOL

philipto committed:
Merge pull request #39631 from Aminev9/master

Pull request accepted.

stablephisher committed:
Add KPRIT institute information

Add KPRIT institute information

stablephisher committed:
Merge branch 'JetBrains:master' into master

SagarTalagatti committed:
Add KLS Gogte Institute of Technology to JetBrains Education list

Kindly add KLS Gogte Institute of Technology, Belagavi, Karnataka, India to the JetBrains Education List

SagarTalagatti committed:
Merge pull request #1 from SagarTalagatti/SagarTalagatti-patch-1

Add KLS Gogte Institute of Technology to JetBrains Education list

philipto committed:
Merge pull request #39639 from SagarTalagatti/master

Pull request accepted.

philipto committed:
Merge pull request #39632 from stablephisher/master

Pull request accepted.

yogicamo committed:
Added Aylesbury Grammar School

philipto committed:
Merge pull request #39641 from yogicamo/master

Pull request accepted.

sbayfield09 committed:
Add files via upload

sai-karumuri committed:
git commit -m "added Chengdu Institute of Biology" to the domain library.

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Merge pull request #39659 from sai-karumuri/master

Pull request accepted.

philipto committed:
Merge pull request #39658 from sbayfield09/master

Pull request accepted.

yardinofa-loop committed:
Add files via upload

philipto committed:
Merge pull request #39663 from yardinofa-loop/master

Pull request accepted.

SukumarPavan committed:
Add files via upload

tomK8 committed:
Create harriswestminstersixthform.txt

philipto committed:
Institución Universitaria de Barranquilla

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Strathmore University

philipto committed:
Merge pull request #39672 from tomK8/patch-1

Pull request accepted.

philipto committed:
Merge pull request #39670 from SukumarPavan/master

Pull request accepted.

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

muqiudeyu committed:
Add Chinese name for Henan Polytechnic University

Add Chinese name for Henan Polytechnic University

lordfrancis committed:
Add cmcbohol.edu.ph doman

For Carmen Municipal College

philipto committed:
Merge pull request #39505 from Aniketh896/master

Added Indian Institute of Management Sirmaur

ahaydarli committed:
Add atmu.txt for Azerbaijan Tourism and Management University

philipto committed:
Rename lib/domains/pcm.edu.np to lib/domains/np/edu/pcm.txt

philipto committed:
Merge pull request #39695 from ahaydarli/master

Pull request accepted.

philipto committed:
Merge pull request #39693 from lordfrancis/master

Pull request accepted.

philipto committed:
Merge pull request #39692 from muqiudeyu/master

Pull request accepted.

philipto committed:
Merge pull request #39329 from sujitchhetri/add-pcm-domain-1

Pull request accepted.

MohammadQaseemQiam committed:
Delete lib/domains/af/edu/kabulpolytechnicuniversity.txt

MohammadQaseemQiam committed:
Add files via upload

philipto committed:
Merge pull request #39697 from MohammadQaseemQiam/master

Pull request accepted.

philipto committed:
Merge pull request #11305 from aallahyar/patch-1

Adding hubrecht.eu domain as a student

Commit:
Add my school domain

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Gdansk University of Technology

philipto committed:
Merge pull request #39709 from science20012003/master

Pull request accepted.

Fercho5656 committed:
Add arandas.txt with institution name

philipto committed:
Merge pull request #39713 from Fercho5656/master

Pull request accepted.

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Fraud

sudoba222 committed:
Add Dushanbe International University domain

WilsonCastro1988 committed:
Create flacso.txt

philipto committed:
Fraud

philipto committed:
1. Facultad de Estudios Superiores Aragón UNAM
2. Cleanup

philipto committed:
Fraud detected

mi260561 committed:
Add files via upload

bhargav2006 committed:
Add files via upload

philipto committed:
Merge pull request #39723 from bhargav2006/master

Pull request accepted.

philipto committed:
Merge pull request #39722 from mi260561/mi260561-patch-1

Pull request accepted.

philipto committed:
Merge pull request #39718 from WilsonCastro1988/feature/flacso-ecuador

Pull request accepted.

philipto committed:
Merge pull request #39717 from sudoba222/master

Pull request accepted.

philipto committed:
Đại học Công nghiệp Thành phố Hồ Chí Minh

philipto committed:
Merge remote-tracking branch 'origin/master'

nat2tk committed:
Add tupr domain

Add Triamudomsuksapattanakarn Ratchada school

philipto committed:
Fraud

philipto committed:
Fraud

philipto committed:
Merge pull request #39725 from nat2tk/master

Pull request accepted.

dbimortals committed:
Add A. C. Patil College of Engineering to acpce.txt

farahive committed:
Create uinsaizu.txt

joaovitorclark committed:
Add Faculdade Delmiro Gouveia (faculdadefdg.edu.br)

philipto committed:
Merge pull request #39738 from joaovitorclark/master

Pull request accepted.

philipto committed:
Merge pull request #39736 from farahive/master

Pull request accepted.

philipto committed:
Merge pull request #39733 from dbimortals/master

Pull request accepted.

davehanma committed:
Create digitechuniversity.txt

giacomoarru committed:
Adding uniolbia.it

philipto committed:
Merge pull request #39747 from giacomoarru/master

Pull request accepted.

philipto committed:
Merge pull request #39745 from davehanma/patch-1

Pull request accepted.

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Fraud, coordinated attack.

Vossi1511 committed:
Add Mariengymnasium-Warendorf

Offical Website: https://www.mariengymnasium.org
Page with course offering: https://mariengymnasium.org/informatik/
Page with proof of domain (§8): https://mariengymnasium.org/wp-content/uploads/2025/11/nutzerordnung_neu_2021.pdf

philipto committed:
Fraud

philipto committed:
Fraud

Mdsaleh99 committed:
Add St. Aloysius College to edu domain

philipto committed:
Renaming file according the repository rules

philipto committed:
Merge pull request #39763 from Mdsaleh99/master

Pull request accepted.

xxooxannn committed:
Add files via upload

mikeywikeytrikey committed:
Add Victor Central School District to students.txt

mikeywikeytrikey committed:
add *.victorschools.org

mikeywikeytrikey committed:
Delete lib/domains/org/victorschools directory

rolando-mr committed:
Add files via upload

colegiohebron committed:
Add files via upload

philipto committed:
Merge pull request #39776 from colegiohebron/master

Pull request accepted.

philipto committed:
Merge pull request #39775 from rolando-mr/master

Pull request accepted.

philipto committed:
Merge pull request #39770 from mikeywikeytrikey/master

Pull request accepted.

philipto committed:
Merge pull request #39765 from Vossi1511/master

Pull request accepted.

xxooxannn committed:
Create rosybuds.txt

xxooxannn committed:
Delete lib/domains/ng/edu/rosybuds.txt

philipto committed:
Merge pull request #39780 from xxooxannn/master

Pull request accepted.

rchoquet42 committed:
add 42 belgium

Commit:
add csust.txt

MamiiAki committed:
Add files via upload

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

shellord committed:
added ekc domain

philipto committed:
Merge pull request #39791 from shellord/master

Pull request accepted.

philipto committed:
Merge pull request #39790 from MamiiAki/master

Pull request accepted.

philipto committed:
Merge pull request #39788 from Justin801227/add-csust-domain

Pull request accepted.

philipto committed:
Merge pull request #39783 from rchoquet42/master

Pull request accepted.

GSriCharanReddy committed:
Add vjit.ac.in domain to vjit.txt

philipto committed:
Merge pull request #39792 from GSriCharanReddy/patch-1

Pull request accepted.

fmoralesaliaga committed:
Create cftestatalaricayparinacota.txt

fmoralesaliaga committed:
Update cftestatalaricayparinacota.txt

burkesquires committed:
Create uttyler.txt

resschrishabhjaiswal-isi committed:
Create igntu.txt

INDIRA GANDHI NATIONAL TRIBAL UNIVERSITY
INDIRA GANDHI NATIOANL TRIBAL UNIVERSITY

philipto committed:
Merge pull request #39797 from resschrishabhjaiswal-isi/master

Pull request accepted.

philipto committed:
Merge pull request #39795 from burkesquires/master

Pull request accepted.

philipto committed:
Merge pull request #39794 from fmoralesaliaga/master

Pull request accepted.

Taku315o committed:
Add Senshu University domain

Taku315o committed:
Merge pull request #1 from Taku315o/add-senshu-university-domain

Add Senshu University domain

ViOLET-ROSE committed:
Add school name

philipto committed:
Merge pull request #39805 from ViOLET-ROSE/master

Pull request accepted.

philipto committed:
Merge pull request #39802 from Taku315o/master

Pull request accepted.

NeddyAP committed:
Add unida.ac.id

NeddyAP committed:
Merge branch 'JetBrains:master' into master

philipto committed:
Cleanup

philipto committed:
Merge remote-tracking branch 'origin/master'

upnch committed:
Request to add school domains

I am writing to formally request educational whitelisting for Université Publique du Nord au Cap-Haïtien (UPNCH).

We deeply understand that education knows no borders, and quality resources are crucial for the growth of students worldwide. Although Haiti is a small country, we share the same mission and passion to provide better education for our youth.

Therefore, I earnestly request that you consider providing our school with an education discount program, helping us access your company's high-quality products at an affordable cost. This support will directly contribute to Haitian students gaining more equal learning opportunities and illuminating their future.

Our school's domain email addresses are @upnch.net and @mail.upnch.net, and we would like to add them to the whitelist.   This is the address of the school in the global higher education database: https://www.whed.net/institutions/IAU-028248

philipto committed:
Merge pull request #39809 from upnch/master

Pull request accepted.

philipto committed:
Merge pull request #39806 from NeddyAP/master

Pull request accepted.

Flower-is-programming committed:
Added a school

philipto committed:
Merge pull request #39812 from Flower-is-programming/master

Pull request accepted.

karol2jag committed:
Add school details for 2lochojnice

philipto committed:
Merge pull request #39818 from karol2jag/master

Pull request accepted.

philipto committed:
Merge pull request #39819 from abdisa19/master

Pull request accepted.

Yukino577 committed:
Add mju.txt

philipto committed:
Merge pull request #39824 from Yukino577/master

Pull request accepted.

harmfulladvocado committed:
added Rhizo

philipto committed:
Merge pull request #39830 from harmfulladvocado/master

Pull request accepted.

fayuth committed:
Create uc

husol committed:
Add files via upload

sato-yasuhiro committed:
Add OECU to domains

philipto committed:
Merge pull request #39838 from sato-yasuhiro/master

Pull request accepted.

philipto committed:
Merge pull request #39834 from husol/master

Pull request accepted.

philipto committed:
Renaming file according the repository rules

philipto committed:
Merge pull request #39831 from fayuth/patch-1

Pull request accepted.

philipto committed:
Cleanup

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Cleanup

philipto committed:
Fraud

Preyansh1 committed:
Add files via upload

kawaiishay committed:
Add Mary Baldwin University domain

kawaiishay committed:
Add Mary Baldwin University domain

philipto committed:
Renaming file according the repository rules

philipto committed:
Merge pull request #39850 from kawaiishay/master

Pull request accepted.

philipto committed:
Merge pull request #39848 from Preyansh1/master

Pull request accepted.

villat committed:
Added INSR college

emman-infinity committed:
Add files via upload

WijayaSaputra-867 committed:
Add SMK TI DWIGUNA domain

philipto committed:
Merge pull request #39856 from WijayaSaputra-867/master

Pull request accepted.

philipto committed:
Renaming file according the repository rules

philipto committed:
Merge pull request #39855 from emman-infinity/master

Pull request accepted.

uzzalkar-mgi committed:
Add AKPI Bangladesh domain

philipto committed:
Merge pull request #39852 from villat/master

Pull request accepted.

philipto committed:
1. The Islamia University of Bahawalpur
2. Reorganized stoplist.txt

philipto committed:
Merge remote-tracking branch 'origin/master'

gummy-smiley committed:
add Rajinibon School

gummy-smiley committed:
Merge pull request #1 from gummy-smiley/rajinibon-school

add Rajinibon School

philipto committed:
Merge pull request #39863 from gummy-smiley/master

Pull request accepted.

philipto committed:
Merge pull request #39858 from uzzalkar-mgi/master

Pull request accepted.

NeuTroNBZh committed:
Add files via upload

Rayo-Dev10 committed:
Add files via upload

Añadir la Institución Universitaria Oriente de Caldas

philipto committed:
Cleanup

philipto committed:
Merge remote-tracking branch 'origin/master'

Feuerball500 committed:
Add files via upload

Please add my School.

philipto committed:
Merge pull request #39872 from Feuerball500/master

Pull request accepted.

philipto committed:
Merge pull request #39871 from Rayo-Dev10/master

Pull request accepted.

philipto committed:
Merge pull request #39870 from NeuTroNBZh/master

Pull request accepted.

shreesthapit1 committed:
Add lacm.edu.np

Add lacm.edu.np

philipto committed:
Merge pull request #39882 from shreesthapit1/master

Pull request accepted.

marcospestana committed:
Add files via upload

marcospestana committed:
Delete file.zip

marcospestana committed:
Add files via upload

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Cleanup

philipto committed:
Fraud

philipto committed:
Merge pull request #39888 from marcospestana/master

Pull request accepted.

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Fraud

philipto committed:
Fraud

Commit:
Add Savannah State University

philipto committed:
Merge pull request #40619 from orozcodiaz/add-savannahstate.edu

Pull request accepted.

philipto committed:
Merge pull request #40618 from orozcodiaz/add-carteret.edu

Pull request accepted.

philipto committed:
Merge pull request #40617 from orozcodiaz/add-curry.edu

Pull request accepted.

oltionfociro-hub committed:
Create fgjm.txt

Adding fgjm.edu.al as domain

oltionfociro-hub committed:
Update fgjm.txt

philipto committed:
Merge pull request #40628 from oltionfociro-hub/master

Pull request accepted.

Cian-X committed:
Add Universitas Yatsi Madani

notrayday committed:
Add iisertirupati.txt with institution name

philipto committed:
Merge pull request #40632 from notrayday/master

Pull request accepted.

philipto committed:
Merge pull request #40630 from Cian-X/add-uym-domain

Pull request accepted.

anonymus-netizien committed:
Add Joginpally B.R. Engineering College (jbrec.edu.in)

philipto committed:
Merge pull request #40634 from anonymus-netizien/add-jbrec-domain

Pull request accepted.

philipto committed:
Hashemite University
Università di Genova

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
fraud

philipto committed:
fraud

philipto committed:
Merge pull request #39089 from kawa11/master

Add UPAL

kawa11 committed:
Отгонтэнгэр Их Сургууль

philipto committed:
1. OSTİM Teknik Üniversitesi - OK
2. Fraud detected

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Merge pull request #40643 from kawa11/master

Pull request accepted.

philipto committed:
Merge pull request #38810 from Hi-c007/master

Add Domain wnu.txt The school is System Technology Institute West Negros University

philipto committed:
Fraud detected

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Fraud detected

souvikadh committed:
Add cemk.ac.in domain for College of Engineering & Management, Kolaghat

philipto committed:
Merge pull request #40658 from souvikadh/master

Pull request accepted.

philipto committed:
Fraud detected

philipto committed:
Merge remote-tracking branch 'origin/master'

manojoli667 committed:
Add ugcnepal.edu.np domain

student200407 committed:
Create beykent

student200407 committed:
Delete lib/domains/edu/beykent

student200407 committed:
Create student.txt

student200407 committed:
Delete lib/domains/tr/edu/beykent directory

student200407 committed:
Create student.txt

İstanbul Beykent Üniversitesi

https://www.beykent.edu.tr/

philipto committed:
Merge pull request #40662 from student200407/master

Pull request accepted.

philipto committed:
Merge pull request #40661 from manojoli667/master

Pull request accepted.

sushanttverma committed:
Add G H Raisoni Skill Tech University (ghrstu.edu.in)

philipto committed:
Merge pull request #40664 from sushanttverma/add-ghrstu-edu-in

Pull request accepted.

philipto committed:
Fraud detected

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
1. University of Buenos Aires added
2. Some new fraud detected

cys-1408 committed:
Add SRM Madurai College for Engineering and Technology

philipto committed:
Merge pull request #40668 from Yuvan14082006/add-srmmcet-domain

Pull request accepted.

philipto committed:
Imberhorne School

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Cleanup

philipto committed:
Dartmouth College

philipto committed:
Madhav Institute of Technology & Science, Gwalior

philipto committed:
Universidade Estácio de Sá (UNESA)

AaronTide committed:
Create puc.txt

mhmdazkanfl committed:
Add SMAN 15 Bandar Lampung to school list

philipto committed:
Merge pull request #40677 from AaronTide/master

Pull request accepted.

philipto committed:
Merge pull request #40676 from mhmdazkanfl/master

Pull request accepted.

amarcrj committed:
Add ITM University Raipur (itmuniversity.org)

philipto committed:
Merge pull request #40679 from amarcrj/master

Pull request accepted.

NeelChotai committed:
Add missing institution name for student.beykent.edu.tr

NeelChotai committed:
Add missing institution name for mipa.idu.ac.id

philipto committed:
Cleanup; also some frauds detected

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Fraud

philipto committed:
Merge pull request #40683 from NeelChotai/add-name-mipa-idu-ac-id

Pull request accepted.

philipto committed:
Merge pull request #40682 from NeelChotai/add-name-student-beykent-edu-tr

Pull request accepted.

philipto committed:
Merge pull request #38968 from Artx999/master

Add Østfold County Municipality

philipto committed:
University of Tripoli

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Mass fraud

navihr committed:
Create psgrkcw.txt

PSGR Krishnammal College for Women added to domain

navihr committed:
Merge pull request #1 from navihr/PSGR-domain-added

Create psgrkcw.txt

philipto committed:
Merge pull request #40690 from navihr/master

Pull request accepted.

BABAR-TAHSEEN55 committed:
create mjcet

BABAR-TAHSEEN55 committed:
Add Mjcollege

philipto committed:
Merge pull request #40694 from BABAR-TAHSEEN55/master

Pull request accepted.

madebysaira committed:
Add SSMS College of Architecture Solapur

philipto committed:
Merge pull request #40700 from madebysaira/add-ssems-solapur

Pull request accepted.

odewaeleislt committed:
Add Institut Saint-Luc Tournai

philipto committed:
Merge pull request #40702 from odewaeleislt/master

Pull request accepted.

levanzed committed:
Add wmocollege.ac.in

## Summary

This pull request adds the official academic email domain for WMO Arts and Science College, Muttil, Kerala, India.

### Domain

- `wmocollege.ac.in`

### Official website

https://www.wmocollege.ac.in/

### Evidence

- The institution uses `@wmocollege.ac.in` as its official institutional email domain.
- Official website: https://www.wmocollege.ac.in/
- Undergraduate programmes: https://wmocollege.ac.in/under-graduate-programme
- Department of Computer Science: https://wmocollege.ac.in/department/computer-science

The institution offers long-term undergraduate degree programmes in computing, including **Bachelor of Computer Application (BCA)** and **Bachelor of Computer Application (Honours)**. The Department of Computer Science provides the BCA programme with a curriculum covering programming, operating systems, computer networks, databases, artificial intelligence, machine learning, internships, and project work.

philipto committed:
Merge pull request #40703 from levanzed/master

Pull request accepted.

philipto committed:
Cleanup

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Al Ehsan Academy

indie-dog committed:
Add Haberdashers' Elstree Schools

philipto committed:
1. 서울과학기술대학교
2. Cleanup

philipto committed:
cmb.ac.lk reclassification

philipto committed:
Domain name change for th-mannheim.de

Peixotim committed:
feat: Add sufix UCB

philipto committed:
Merge pull request #40711 from Peixotim/fix-ucb-catolica-domain

Pull request accepted.

philipto committed:
Merge pull request #40709 from indie-dog/patch-3

Pull request accepted.

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

bao826078-glitch committed:
Add Son La College domain file

philipto committed:
Renaming file according the repository rules

bao826078-glitch committed:
Delete lib/domains/vn/edu/cdsonla/edu directory

bao826078-glitch committed:
Add cdsonla.txt with Son La College name

philipto committed:
Cleanup

philipto committed:
Fraud

scintillating2107 committed:
Added Rajkiya Engineering College kannauj

philipto committed:
Cleanup

philipto committed:
Merge pull request #40717 from scintillating2107/master

Pull request accepted.

philipto committed:
Merge pull request #40716 from bao826078-glitch/master

Pull request accepted.

philipto committed:
1. Son La Community College
2. Removing some frauds

philipto committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	lib/domains/vn/edu/cdsonla.txt

philipto committed:
Fraud

philipto committed:
Fraud

philipto committed:
Fraud

philipto committed:
Fraud

philipto committed:
Fraud

idwa111 committed:
cbtis130

philipto committed:
Merge pull request #40719 from idwa111/master

Pull request accepted.

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

adham-khalill committed:
Add AAST information to aast.txt

hemrajiscool committed:
Update gcoej.txt with college name

philipto committed:
Merge pull request #40722 from hemrajiscool/master

Pull request accepted.

philipto committed:
Merge pull request #40721 from adham-khalill/add-aast-edu

Pull request accepted.

MirAyaanAli committed:
Added Lords college

webuyibuy committed:
Add T. John Group of Institutions (tjohngroup.com)

MirAyaanAli committed:
Merge branch 'JetBrains:master' into swot-updated

sauhardoroy committed:
added Indian Institute of Management Bangalore

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Merge pull request #40727 from MirAyaanAli/swot-updated

Pull request accepted.

philipto committed:
Merge pull request #40726 from sauhardoroy/master

Pull request accepted.

philipto committed:
Merge pull request #40724 from webuyibuy/add-tjohngroup-domain

Pull request accepted.

LEON-72 committed:
Add Osaka Institute of Technology

PechenkiUA committed:
Create unu.txt

killer90234 committed:
Add Shreyarth University to stanford.txt

philipto committed:
Merge pull request #40739 from killer90234/patch-1

Pull request accepted.

philipto committed:
Merge pull request #40738 from PechenkiUA/patch-2

Pull request accepted.

philipto committed:
Merge pull request #40736 from LEON-72/master

Pull request accepted.

Harshal-S-Bhoyar committed:
Add pcenagpur.eud.in domain

Sytten committed:
Remove unrelated university to Stanford

philipto committed:
Fraud, cleanup

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Merge pull request #40746 from Sytten/ef/fix-standford-abuse

Pull request accepted.

philipto committed:
Merge pull request #40744 from Harshal-S-Bhoyar/patch-1

Pull request accepted.

Mozes2024 committed:
Add Santa Fe College to the list of institutions

philipto committed:
Merge pull request #40749 from Mozes2024/patch-1

Pull request accepted.

Akashmishra1209 committed:
Added Baltimore City Community College

Hermuc committed:
Add ncwu.edu.cn for North China University of Water Resources and Electric Power

This PR adds the domain ncwu.edu.cn for North China University of Water Resources and Electric Power. The domain is used by students and faculty (e.g., stu.ncwu.edu.cn). It is not in the stoplist or abuse list. Please review and merge. Thank you.

philipto committed:
Merge pull request #40757 from Hermuc/add-ncwu

Pull request accepted.

philipto committed:
Merge pull request #40755 from Akashmishra1209/master

Pull request accepted.

EasternVibez committed:
Add Blue Ridge Community and Technical College

philipto committed:
Merge pull request #40758 from EasternVibez/EasternVibez-patch-1

Pull request accepted.

Rithwik-P-26 committed:
Add IIT Delhi - Abu Dhabi information file

initial fork seems to have been rejected. this is 2nd try to add my uni domain to the list

siddhesh-padval committed:
Add University of Mumbai to mu.txt

SathiyaSenpai committed:
Add hicas.ac.in domain

philipto committed:
Merge pull request #40764 from SathiyaSenpai/master

Pull request accepted.

philipto committed:
Merge pull request #40763 from siddhesh-padval/master

Pull request accepted.

philipto committed:
Merge pull request #40762 from Rithwik-P-26/patch-1

Pull request accepted.

nikitayxp committed:
Add new file espa.txt with school grouping information

SaiyamSoni committed:
Add IITRAM (iitram.ac.in)

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Merge pull request #40767 from SaiyamSoni/add-iitram

Pull request accepted.

philipto committed:
Merge pull request #40766 from nikitayxp/add-espa-edu-pt

Pull request accepted.

BeardBishwo committed:
Add Nepal Polytechnic Institute (NPI)

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

elementaryrock committed:
Add Marian Engineering College

philipto committed:
Fraud

philipto committed:
Fraud

vinodff committed:
Add bvcits.edu.in (Bonam Venkata Chalamayya Institute of Technology & Science)

philipto committed:
Merge pull request #40773 from vinodff/master

Pull request accepted.

philipto committed:
Merge pull request #40772 from elementaryrock/add-marian-engineering-college

Pull request accepted.

philipto committed:
Merge pull request #40770 from BeardBishwo/add-nepal-polytechnic-institute

Pull request accepted.

seanvishwakarma committed:
Add R. D. National College information

philipto committed:
Non-eligible domain

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Non-eligible domain

philipto committed:
Merge pull request #40779 from seanvishwakarma/master

Pull request accepted.

philipto committed:
Non-eligible domain

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Fraud

Hermuc committed:
Delete lib/domains/cn/edu/ncwu.txt

jorian101 committed:
Add emi.edu.bo - Escuela Militar de Ingeniería

philipto committed:
Merge pull request #40788 from jorian101/add-emi-edu-bo

Pull request accepted.

philipto committed:
Merge pull request #40787 from Hermuc/master

Pull request accepted.

WilsonRamropui committed:
Add Manipur Technical University to mtu.txt

shivanshshekhar11 committed:
Added Ambala College of Engineering and Applied Research

saravanan2736 committed:
Add files via upload

projsMED committed:
Add Universidade Federal do Norte do Tocantins

philipto committed:
Merge pull request #40797 from projsMED/add-ufnt-domain

Pull request accepted.

philipto committed:
Merge pull request #40792 from saravanan2736/master

Pull request accepted.

philipto committed:
Merge pull request #40790 from shivanshshekhar11/master

Pull request accepted.

philipto committed:
Merge pull request #40789 from WilsonRamropui/master

Pull request accepted.

Sytten committed:
Remove fraud for stanford

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Fraud

philipto committed:
Fraud

philipto committed:
Fraud

Squidz5 committed:
Add bbchs.org school domain

philipto committed:
Merge pull request #40801 from Squidz5/master

Pull request accepted.

philipto committed:
Merge pull request #40798 from Sytten/ef/stanford-fraud

Pull request accepted.

kawa11 committed:
Add 서정대학교

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Non-eligible domain

philipto committed:
Merge pull request #40805 from kawa11/master

Pull request accepted.

Artemis43 committed:
Add East Point Group of Institutions details

kawa11 committed:
Add Universidad Interamericana to lainter.txt

philipto committed:
Non-eligible domains

philipto committed:
Merge remote-tracking branch 'origin/master'

prusson committed:
Create mhs.txt

philipto committed:
Merge pull request #40812 from prusson/master

Pull request accepted.

philipto committed:
Merge pull request #40808 from Artemis43/master

Pull request accepted.

philipto committed:
Merge pull request #40807 from kawa11/master

Pull request accepted.

kawa11 committed:
วิทยาลัยเทคนิคพระนครศรีอยุธยา

philipto committed:
Merge pull request #40813 from kawa11/master

Pull request accepted.

TayfurYldz committed:
Add Yüksek İhtisas Üniversitesi

kawa11 committed:
Add ceis.txt

philipto committed:
Merge pull request #40815 from kawa11/master

Pull request accepted.

philipto committed:
Merge pull request #40814 from TayfurYldz/add-yuksek-ihtisas-university

Pull request accepted.

kawa11 committed:
Add Centro Universitario Interamericano

subhom2003 committed:
Create jncasr.txt

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Fraud

philipto committed:
Fraud

philipto committed:
Merge pull request #40817 from subhom2003/master

Pull request accepted.

philipto committed:
Merge pull request #40816 from kawa11/master

Pull request accepted.

kawa11 committed:
Add Centro Universitario Metropolitano Hidalgo

philipto committed:
Merge pull request #40818 from kawa11/master

Pull request accepted.

Nanthagopal18 committed:
Add Alpha Group of Institutions to domains list

Karen64345 committed:
Add Mid-Valley International College to domains

philipto committed:
Merge pull request #40824 from Karen64345/master

Pull request accepted.

Nanthagopal18 committed:
Add Alpha Group of Institutions to alphagroup.txt

Nanthagopal18 committed:
Delete lib/domains/in/edu/alphagroup.txt

philipto committed:
University of Human Environments (人間環境大学)

philipto committed:
Merge remote-tracking branch 'origin/master'

Anaskhan04 committed:
Add Rajkiya Engineering College Banda to domains

philipto committed:
Merge pull request #40827 from Anaskhan04/master

Pull request accepted.

philipto committed:
Merge pull request #40825 from Nanthagopal18/master

Pull request accepted.

kawa11 committed:
Add nrru.txt with university name

Vasanth-Builds committed:
Add RMK College of Engineering and Technology (rmkcet.ac.in)

philipto committed:
Merge pull request #40832 from VasanthSaravanann/master

Pull request accepted.

philipto committed:
Merge pull request #39220 from ericgskim/master

Please register this school to the domain.

philipto committed:
Merge pull request #39215 from Vientos6/master

Add Universidad Del Valle de Matatipac

kawa11 committed:
Merge branch 'JetBrains:master' into master

kawa11 committed:
Add

kawa11 committed:
Delete lib/domains/th/ac/nrru.txt

philipto committed:
Merge pull request #39194 from mohanadkalal/master

https://ar.cetb.edu.ly

philipto committed:
Merge pull request #39146 from ysochoa-create/master

Agrego archivo de licencia estudiantil Asturias

philipto committed:
Merge pull request #39137 from Dioumadev221/patch-1

Add ISEP-AT institution information

philipto committed:
Merge pull request #40838 from kawa11/master

add

philipto committed:
King's College, HK

philipto committed:
Merge remote-tracking branch 'origin/master'
